### PR TITLE
return expanded user when getGroupMembers

### PR DESCRIPTION
**Pull request by mongolianprincess**
expanded result of getGroupMembers to include user

### CODE REVIEW

File: api/group/index.js [hunk 1]
```diff
@@ -170,10 +170,19 @@ module.exports = function (api) {
...
+        async.mapSeries(result, function (member, cb) {
+          api.user.mapUserIdToUser(keyspace, member, ['user'], null, function (err, memberWithUser) {
+            if (err) { return cb(err); }
+            cb(null, memberWithUser);
+          });
+        }, function (err, membersWithUser) {
+          if (err) { return next(err); }
+          next(null, membersWithUser);
+        });
+      } else { return next(api.common.error(404, 'Unable to find group members: ' + group)); }
```
Comment (Jae) [inline api/group/index.js]: tell me I'm an old school but I believe empty group members don't warrant an error condition of 404

File: api/group/index.js [hunk 2]
```diff
@@ -170,10 +170,19 @@ module.exports = function (api) {
...
 
   function getGroupMembers (keyspace, group, next) {
-    client.get(q(keyspace, 'selectGroupMembers'), [group], {}, function (err, result) {
+    client.execute(q(keyspace, 'selectGroupMembers'), [group], {}, function (err, result) {
       if (err) { return next(err); }
-      if (!result) { return next(api.common.error(404, 'Unable to find group members: ' + group)); }
-      next(null, result);
+      if (result && result.length > 0) {
+        async.mapSeries(result, function (member, cb) {
+          api.user.mapUserIdToUser(keyspace, member, ['user'], null, function (err, memberWithUser) {
```
Comment (Jae) [inline api/group/index.js]: api.user.mapUserIdToUser(keyspace, member, ['user'], null, cb); would do the same as what you've got

File: api/group/index.js [hunk 3]
```diff
@@ -170,10 +170,19 @@ module.exports = function (api) {
...
+            if (err) { return cb(err); }
+            cb(null, memberWithUser);
+          });
+        }, function (err, membersWithUser) {
```
Comment (Jae) [inline api/group/index.js]: again perhaps , next); would do the same here

File: api/group/index.js (hunk 1 above)
Comment (mongolianprincess) [reply api/group/index.js]: agreed, changed to simply throwing an error

File: api/group/index.js (hunk 2 above)
Comment (mongolianprincess) [reply api/group/index.js]: done

File: api/group/index.js (hunk 3 above)
Comment (mongolianprincess) [reply api/group/index.js]: done

**Pull request merged**